modin/core/execution/utils.py

```diff
@@ -29,3 +31,45 @@ def set_env(**environ):
     finally:
         os.environ.clear()
         os.environ.update(old_environ)
+
+
+# Check if the function already exists to avoid circular imports
```

Comment:
We just add a feature in this PR and don't use anywhere for now, but it is going to be used in case_when, right?

Reply:
Right, this PR just only adds the functionality.